<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellMain</class>
 <widget class="QWidget" name="CellMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>聊天主界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* 统一背景与字体 */
QWidget {
    background-color: #f5f5f5;
    font-family: "Microsoft YaHei";
    font-size: 12px;
}

/* 左侧工具按钮 */
QToolButton {
    border: none;
    background-color: transparent;
    padding: 10px;
    margin: 5px;
    icon-size: 28px;
}
QToolButton:hover {
    background-color: #e6e6e6;
    border-radius: 6px;
}

/* 好友列表 */
QListWidget {
    border: none;
    background: white;
}
QListWidget::item {
    padding: 8px;
}
QListWidget::item:selected {
    background-color: #e6f7ff;
    border-radius: 5px;
}

/* 输入框样式 */
QInputDialog {
    background-color: #ffffff;
    border-radius: 8px;
}
QInputDialog QLineEdit {
    padding: 5px;
    border: 1px solid #ccc;
    border-radius: 4px;
}
   </string>
  </property>

  <!-- 左侧导航区域 -->
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>672</height>
    </rect>
   </property>

   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect><x>10</x><y>10</y><width>80</width><height>20</height></rect>
    </property>
    <property name="text"><string>当前账号：</string></property>
   </widget>

   <widget class="QLabel" name="account">
    <property name="geometry">
     <rect><x>10</x><y>30</y><width>80</width><height>20</height></rect>
    </property>
    <property name="text"><string>账号</string></property>
   </widget>

   <widget class="QLabel" name="headerl">
    <property name="geometry">
     <rect><x>20</x><y>60</y><width>60</width><height>60</height></rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px; border:1px solid #ccc;</string>
    </property>
    <property name="text"><string>头像</string></property>
   </widget>

   <widget class="QToolButton" name="addFriend">
    <property name="geometry">
     <rect><x>20</x><y>140</y><width>60</width><height>60</height></rect>
    </property>
    <property name="toolTip"><string>添加好友</string></property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/addfriend.png</normaloff>:/addfriend.png</iconset>
    </property>
    <property name="text"><string/></property>
   </widget>

   <widget class="QToolButton" name="deleteFriend">
    <property name="geometry">
     <rect><x>20</x><y>210</y><width>60</width><height>60</height></rect>
    </property>
    <property name="toolTip"><string>删除好友</string></property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/deleteFriend.png</normaloff>:/deleteFriend.png</iconset>
    </property>
    <property name="text"><string/></property>
   </widget>

   <widget class="QToolButton" name="applyFriend">
    <property name="geometry">
     <rect><x>20</x><y>280</y><width>60</width><height>60</height></rect>
    </property>
    <property name="toolTip"><string>修改密码</string></property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/applyFriend.png</normaloff>:/applyFriend.png</iconset>
    </property>
    <property name="text"><string/></property>
   </widget>

   <widget class="QToolButton" name="friendsitem">
    <property name="geometry">
     <rect><x>20</x><y>350</y><width>60</width><height>60</height></rect>
    </property>
    <property name="toolTip"><string>朋友圈</string></property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/wechatfriend.png</normaloff>:/wechatfriend.png</iconset>
    </property>
    <property name="text"><string/></property>
   </widget>
  </widget>

  <!-- 好友列表 -->
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect><x>100</x><y>0</y><width>200</width><height>672</height></rect>
   </property>
  </widget>

  <!-- 聊天窗口区域 -->
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect><x>300</x><y>0</y><width>699</width><height>672</height></rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>

 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
